canvasapi/account.py

```diff
@@ -1039,6 +1040,74 @@ def get_all_outcome_links_in_context(self):
             'accounts/{}/outcome_group_links'.format(self.id)
         )
 
+    def add_grading_standards(self, title, grading_scheme_entry, **kwargs):
+        """
+        Creates a new grading standard for the account.
+
+        :calls `POST /api/v1/accounts/:account_id/grading_standards \
+        <https://canvas.instructure.com/doc/api/grading_standards.html#method.grading_standards_api.create>`
+
+        :param title: The title for the Grading Standard
+        :type title: str
+        :param grading_scheme: A list of dictionaries containing keys for "name" and "value"
+        :type grading_scheme: list[dict]
+        :rtype: :class:`canvasapi.grading_standards.GradingStandard`
+        """
+        if not isinstance(grading_scheme_entry, list) or len(grading_scheme_entry) <= 0:
+            raise ValueError("Param `grading_scheme_entry` must be a non-empty list.")
+
+        for entry in grading_scheme_entry:
+            if not isinstance(entry, dict):
+                raise ValueError("grading_scheme_entry must consist of dictionaries.")
+            if "name" not in entry or "value" not in entry:
+                raise ValueError("Dictionaries with keys 'name' and 'value' are required.")
+        kwargs["grading_scheme_entry"] = grading_scheme_entry
+
+        response = self._requester.request(
+            'POST',
+            'accounts/%s/grading_standards' % (self.id),
+            title=title,
+            _kwargs=combine_kwargs(**kwargs)
+        )
+
+        return GradingStandard(self._requester, response.json())
+
+    def get_grading_standards(self):
```

Comment:
Add `**kwargs` to this and other methods. Although the Canvas documentation doesn't list any arguments, anything returning a paginated list can benefit from the user passing args such as `per_page`. To be safe, add it to all new methods you've created here.

Be sure to pass it along w/ combine_kwargs!